src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/ExportOptions.cs

```diff
@@ -7,7 +7,7 @@ namespace System.Runtime.Serialization
 {
     public class ExportOptions
     {
-        private Collection<Type> _knownTypes;
+        private Collection<Type>? _knownTypes;
```

Comment:
should DataContractSurrogate be nullable? (not guarded against null)

Reply:
SUPPORT_SURROGATE is never defined, AFAIK. I updated it to be `IDataContractSurrogate?`, but the compiler will never see it.